### ISSUE TITLE
Token height indicator

### ISSUE THREAD

**Issue by digitalsadhu** [otfbm/issues#13]
**Is your feature request related to a problem? Please describe.**
Add a way to show on the token, how high off the ground the token is (in ft)

**Describe the solution you'd like**
A flag in a token segment could be used to specify the height. Then perhaps a small bubble indicating the height number could be placed on the token. The usual challenges of small token sizes exist.

**Describe alternatives you've considered**
N/A

**Additional context**
N/A

**Comment by lpduarte** [comment]
Using the corners to illustrate whether the combatant is above or bellow. Adding the height value overloads the token. I think it's enough to show this and have the player check the initiative order list to find out exactly how high the combatant is. We will probably need a legend on the documentation site. Shouldn't be too hard to do though.

![otfbm-height-test-1](https://user-images.githubusercontent.com/16154620/92052303-2bd47a80-ed85-11ea-8708-1c72c1edc1b3.jpg)

**Comment by mjs1** [comment]
what the arrows relative to, ground level? how would multiple elevations work for flying 30', 45', 60' etc
if there is space and it's not interfering with other text wouldn't we want to have `30'` on the icon somewhere?

**Comment by digitalsadhu** [comment]
Could you use colors to indicate flying vs underground vs elevated ground?

**Comment by lpduarte** [comment]
@mjs1 yes, the reference would always be ground level to keep things simple, we'd have to mention that in our rules. Having the token display the height's value seems impossible to read. It sure wouldn't fit inside the corner triangle and still be readable. Besides if we're also putting the generated name on the bottom right corner (not yet designed), and if the same char had a condition, it would be too much info displaying on the same token.

@digitalsadhu This solution is the simplest. He's either above or below ground level. But we can do that, for sure. Was using the brand's blue but we can set a number of colours for different states, just need to figure out which make make. We will do have to add that info to the new legend.
